CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 ## Unreleased
 
+## v0.35.0
+
+## 🛑 Breaking changes 🛑
+
+- Rename configparser.Parser to configparser.ConfigMap (#5070)
+- Rename TelemetryCreateSettings -> TelemetrySettings (#5169)
+
+## 💡 Enhancements 💡
+
+- chore: update influxdb exporter and receiver (#5058)
+- chore(dynatrace): use payload limit from api constants (#5077)
+- Add documentation for filelog's new force_flush_period parameter (#5066)
+- Reuse the gzip reader with a sync.Pool (#5145)
+- Add a trace observer when splunkhecreceiver is used for logs (#5063)
+- Remove usage of deprecated pdata.AttributeValueMapToMap (#5174)
+
+## 🧰 Bug fixes 🧰
+
+- Use staleness markers generated by prometheus, rather than making our own (#5062)
+- \[exporter/datadogexporter] skip NaN and infinite values (#5053)
```

Comment:
```suggestion
- [exporter/datadogexporter] skip NaN and infinite values (#5053)
```

Reply:
not sure if this is important, but the format for `exporter/datadogexporter` differs from previous versions.